saga/sagalogs/file_test.go

```diff
@@ -170,3 +172,54 @@ func TestGetMessages_SagaDoesNotExist(t *testing.T) {
 		t.Errorf("Expeceted no messages to be returned %+v", msgs)
 	}
 }
+
+/*
+	Set different saga update channel size using DefaultSagaUpdateChSize
+	Benchmarked on cpu: Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
+
+	For sagasCount = 1, updatesPerSaga = 1000 and following DefaultSagaUpdateChSize values
+	* DefaultSagaUpdateChSize=1:     1 iteration(s)	20359031637 ns/op  44309080 B/op  60419 allocs/op
+	* DefaultSagaUpdateChSize=10:	 1 iteration(s)  2209322245 ns/op	6610864 B/op  29788 allocs/op
+	* DefaultSagaUpdateChSize=100:	 3 iteration(s)	  368981765 ns/op 	2278261 B/op  23838 allocs/op
+	* DefaultSagaUpdateChSize=1000:	 7 iteration(s)	  174756943 ns/op	1861685 B/op  23132 allocs/op
+
+	For sagasCount = 10, updatesPerSaga = 100 and following DefaultSagaUpdateChSize values
+	* DefaultSagaUpdateChSize=1:     1 iteration(s)	17434411997 ns/op  6995688 B/op  45573 allocs/op
+	* DefaultSagaUpdateChSize=10:	 1 iteration(s)  2553031047 ns/op  2832392 B/op  28176 allocs/op
+	* DefaultSagaUpdateChSize=100:	 2 iteration(s)	  705908630 ns/op  1966188 B/op  23760 allocs/op
+*/
+func BenchmarkProcessUpdatesInFileSagaLog(b *testing.B) {
```

Comment:
I hate to say it but I was reading these for a minute before I realized it was file log - this is good info but we only use file log in testing and don't care about the perf. Just worried about it possibly causing confusion.

Reply:
Sure, since we're not using it anywhere, I'll remove this benchmark.